gamify-impl/src/main/java/ch/heigvd/gamify/domain/endUser/EndUserIdentifier.java

```diff
@@ -0,0 +1,24 @@
+package ch.heigvd.gamify.domain.endUser;
+
+import ch.heigvd.gamify.domain.app.App;
+import java.io.Serializable;
+import javax.persistence.Embeddable;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Builder
+@Embeddable
+public class EndUserIdentifier implements Serializable {
+  private String userId;
+
+  @ManyToOne
```

Comment:
Can we use `@ManyToMany` ? For a single username, we should be able to find multiple apps, and for a single app, we should be able to find multiple usernames.

Reply:
Instead of changing to `@ManyToMany`, it'd be better to test our current behaviour to check if a change is needed or not.

Thanks @alexandrepiveteau for volunteering!

Reply:
Tested in #54.